ethers-core/src/types/chain.rs

```diff
@@ -412,6 +414,10 @@ impl Chain {
                 ("https://blockscout.com/xdai/mainnet/api", "https://blockscout.com/xdai/mainnet")
             }
 
+            ScrollAlphaTestnet => {
+                ("https://blockscout.scroll.io/api", "https://blockscout.scroll.io/")
+            }
```

Comment:
is this right?

kinda weird, because this looks more like the mainnet URL?

Reply:
yeah.. but it is for scroll-alpha-testnet [https://blockscout.scroll.io/](https://blockscout.scroll.io/)

You can also confirm with the scroll official page [here](https://scroll.io/alpha)

Reply:
very cursed imo

will cause issues once they replace it with mainnet or they add a `mainnet` to the path or domain, which would be even more confusing -.-

but looks like this is correct (at least for now), so lgtm